docs/root/_include/ads.yaml

```diff
@@ -0,0 +1,51 @@
+node:
+  # set <cluster identifier>
+  cluster: envoy_cluster
+  # set <node identifier>
+  id: envoy_node
+
+dynamic_resources:
+  ads_config:
+    api_type: GRPC
+    transport_api_version: V3
+    grpc_services:
+    - envoy_grpc:
+        cluster_name: ads_cluster
+  cds_config:
+    resource_api_version: V3
+    ads: {}
+  lds_config:
+    resource_api_version: V3
+    ads: {}
+
+static_resources:
+  clusters:
+  - name: ads_cluster
+    connect_timeout: 5s
+    type: STRICT_DNS
```

Comment:
i had to switch this to `STRICT_DNS` - with `STATIC` + ip adress set - ie the original - it worked locally bringing up the server but in the `config_load_validation` tests it was causing a segfault